jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/inputGroup/InputGroupButtonAddonsTests.java

```diff
@@ -0,0 +1,59 @@
+package io.github.epam.bootstrap.tests.composite.section.inputGroup;
+
+import io.github.epam.TestsInit;
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static com.epam.jdi.light.driver.WebDriverFactory.getDriver;
+import static io.github.com.StaticSite.bsPage;
+import static io.github.com.pages.BootstrapPage.*;
+import static io.github.epam.states.States.shouldBeLoggedIn;
+
+public class InputGroupButtonAddonsTests extends TestsInit {
+    private static final String JS_SCROLL_TO_ELEMENT = "arguments[0].scrollIntoView(true);";
+    private static final String HEADER_RIGHT_PART = "//h4[contains(., 'Input group - Button addons')]";
+    private String text = "XA-XA",
+       placeholder_text = "Recipient's username";
+
+    @BeforeMethod
+    public void before() {
+        shouldBeLoggedIn();
+        bsPage.shouldBeOpened();
+        ((JavascriptExecutor) getDriver()).executeScript(JS_SCROLL_TO_ELEMENT, getDriver().findElement(By.xpath(HEADER_RIGHT_PART)));
```

Comment:
please use jdi class instead of direction selenium calls, jsExecute(). we have own find method instead of selenium findElement, for example  $()